jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -669,7 +670,7 @@ public void setContextPath(String contextPath)
     {
         if (isStarted())
             throw new IllegalStateException(getState());
-        _contextPath = contextPath;
+        _contextPath = URIUtil.normalizePath(contextPath);
```

Comment:
See #8343, as that will change this to canonicalizePath. ... race you to merge :)